py/build.gradle

```diff
@@ -315,7 +317,7 @@ task testPythonMojoPipeline(type: PythonTask, dependsOn: distPython) {
 }
 
 task testPythonREST(type: PythonTask, dependsOn: distPython) {
-    extraArgs(*createUnitTestArgs())
+    extraArgs(*createUnitTestRESTApiArgs())
```

Comment:
I think this is not necessary, the option is already defined here: https://github.com/h2oai/sparkling-water/blob/master/py/tests/unit/with_runtime_clientless_sparkling/conftest.py#L36

Reply:
Good catch, thank you!